main.go

```diff
@@ -48,9 +48,14 @@ func run(c *cli.Context) error {
 	}
 
 	version := c.Args()[0]
+	latest := false
```

Comment:
Could just do this, right?
```suggestion
	latest := c.Bool("latest")
```

Reply:
Oh yeah, that is a bit faster for processing it. Need to remove the related if block in that function so I'll edit it manually. (Didn't know this change suggestion stuff existed, neat feature)